File: src/main/java/jp/itohiro/intellij/sample/domain/StackOverflowItems.java
package jp.itohiro.intellij.sample.domain;

import com.google.api.client.util.Key;

public class StackOverflowItems {
    @Key
    private StackOverflowItem[] items;
    @Key
    private boolean has_more;
    @Key
    private long quota_max;
    @Key
    private long quota_remaining;

    public JsonItem[] getItems() {
        return items;
    }
}

File: src/main/kotlin/jp/itohiro/intellij/searchit/domain/StackOverflowItems.kt
package jp.itohiro.intellij.searchit.domain

import com.google.api.client.util.Key

public class StackOverflowItems {
    @Key
    private var items: Array<StackOverflowItem>? = null
    @Key
    private var has_more: Boolean = false
    @Key
    private var quota_max: Long = 0
    @Key
    private var quota_remaining: Long = 0

    public fun getItems(): Array<out JsonItem>? {
        return items
    }
}
